js/api/Geofencing.js

```diff
@@ -39,12 +37,12 @@ export function clearActiveMap(component) {
  */
 export function setupGeofences(callback) {
```

Comment:
Some iOS biddings seem to be missing after syncing with master due to PR #137 
Will update this shortly.